consoles/vnc_base.pm

```diff
@@ -52,17 +46,15 @@ sub connect_remote {
     return $self->{vnc};
 }
 
-sub request_screen_update {
-    my ($self) = @_;
+sub request_screen_update ($self, @) {
```

Comment:
is there a case where the method is called with an argument?

Reply:
I am not exactly sure but I think so when used as a callback target over the RPC implementation which somehow ends up with more unused, unfilled arguments passed. I might have overdone it in one or two cases but normally I would only add the argument if some tests trigger an error or warning